.github/workflows/release_nightly.yml

```diff
@@ -97,14 +77,21 @@ jobs:
             echo "Version $version does not match nightly tag ${{ inputs.nightly_tag_base }}. Exiting the workflow."
             exit 1
           fi
+          # Strip the leading `v` from the version
+          version=$(echo $version | sed 's/^v//')
+          echo "version=$version" >> $GITHUB_OUTPUT
 
       - name: Build project for distribution
-        run: make build base=true
+        run: make build base=true args="--wheel"
 
       - name: Test CLI
         run: |
-          python -m pip install src/backend/base/dist/*.whl
-          python -m langflow run --host 127.0.0.1 --port 7860 &
+          # TODO: unsure why the whl is not built in src/backend/base
```

Comment:
I have _no idea_ why this behavior changed on me. Between pushes, through no change on my end that I can tell, `make build base` using `uv` started putting the `base` whl into `root/dist` instead of `root/src/backend/base/dist`. This is a hack that moves it back to where everything else expects it to be.